client/src/pages/EspecificConnection.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { Fragment, useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import InputConnection from "../components/Connections/InputConnection";
+import ListConnections from "../components/Connections/ListConnections";
+
+export interface ConnectionInterface{
+    user1_id: number; 
+    user2_id: number; 
+  }
+
+export const EspecificConnections = () => {
+
+    const params = useParams();
+    const [connections, setConnections] = useState<any | any[]>([]);
+
+    const onSubmitForm = (e: any, user1: string, user2: string) => {
+      e.preventDefault();
+      fetch("http://localhost:3000/connections" + "/" + user1 + "/" + user2, {
+        method: "POST",
+        headers: { "Content-Type": "application/json" },
+      })
+      .then(() => { })
+      .catch((reason) => console.error("Create connection promise rejected : " + reason))
+    };
+
+
+
+    const getConnectionsTable = () => {
+
+      fetch("http://localhost:3000/connections-table" + '/' + params.id)
+      .then((res) => {
+        res.json()
+        .then((resJson) => {
+          setConnections(resJson);
+        }).catch((reason) => console.error("res.json() promise rejected : " + reason));
+      }).catch((reason) => console.error("Especific connection promise rejected : " + reason));
+    };
```

Comment:
Have in mind you can cascade the promise resolutions using the returned values. I think it provides a simpler code 

```
Promise
.then(do something)
.then(do something)
.then(do something)
.catch(handle error)
```

Reply:
I think I should do that change in the react-refactor branch because the future merge would cause a conflict between react-design and react-refactor branches on the .then.cath code lines isn't it?

Reply:
okay